packages/components/app/styles/components/table.scss

```diff
@@ -51,11 +51,27 @@ $hds-table-border-color: var(--token-color-border-primary);
       th:first-child {
         border-left: none;
         border-top-left-radius: $hds-table-inner-border-radius;
+
+        button {
+          &:focus,
+          :focus-visible {
+            border-color: var(--token-color-focus-action-external);
+            border-top-left-radius: $hds-table-inner-border-radius;
+          }
+        }
       }
 
       th:last-child {
         border-right: none;
         border-top-right-radius: $hds-table-inner-border-radius;
+
+        button { // I don't like this, it feels weird to have declared in multiple places- open to ideas on how to organize this CSS so we don't have to repeat selectors like this.
```

Comment:
cc @didoo I'm open to some suggestions on this - I am not _quite_ sure how to organize this.